src/Review.hbm.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version = "1.0" encoding = "utf-8"?>
+<!DOCTYPE hibernate-mapping PUBLIC 
+"-//Hibernate/Hibernate Mapping DTD//EN"
+"http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd"> 
+
+<hibernate-mapping>
+   <class name = "com.marketplace.domain.Review" table = "review">
+      
+      <meta attribute = "class-description">
+         This class contains review information. 
+      </meta>
+      
+      <id name = "reviewID" type = "int" column = "review_id">
+         <generator class="increment"/>
+      </id>
+      
+      <many-to-one name="customer" class="com.marketplace.domain.Customer"
+            column="customer_id" unique="true" not-null="true"
+            cascade="none" />
+      <many-to-one name="product" class="com.marketplace.domain.Product"
+            column="product_id" unique="true" not-null="true"
+            cascade="none" />      
+      <property name = "reviewBody" column = "street_address" type = "string"/>
```

Comment:
Missing review score! If you think we should drop score though and just have the actual review, I'm good with that too - will just have to update the DB.

Also `reviewBody` is mapped to `street_address` instead of `review_body`!

Reply:
no problem I'll add review score right now!